tests/analysis/adaptation/test_adaptation.py

```diff
@@ -132,5 +146,30 @@ def test_calculate_bc_ratio_returns_gdf(
             mocked_adaptation.adaptation_collection.adaptation_options
         ):
             assert _result[f"{_option.id}_bc_ratio"].sum(axis=0) == pytest.approx(
-                _nof_rows * (4.0 + i) / (1.0 + i)
+                len(_result.index) * (4.0 + i) / (1.0 + i)
             )
+
+    def test_output_gdf_can_be_exported_to_gpkg(
+        self,
+        valid_adaptation_config_with_input: tuple[
+            AnalysisInputWrapper, AnalysisConfigWrapper
+        ],
+        test_result_param_case: Path,
+    ):
+        # 1. Define test data.
+        _adaptation = Adaptation(
+            valid_adaptation_config_with_input[0], valid_adaptation_config_with_input[1]
+        )
+
+        # 2. Run test.
+        _result = _adaptation.execute().results_collection[0]
+
+        _output_path = _result.output_path
+        _output_path.mkdir(parents=True, exist_ok=True)
+
+        _result.analysis_result.to_file(
+            _result.output_path.joinpath("adaptation_output.gpkg"), driver="GPKG"
+        )
+
+        # 3. Verify expectations.
+        assert test_result_param_case.exists()
```

Comment:
This only checks for the directory, not for the file. In addition you could load the geopandas file and compare it to the result one.
In addition, this step should be done using the analysis result wrapper. Or even further, move this test to the corresponding runner (`AdaptationAnalysisRunner`) as it has both execution and export.

Reply:
Checking the existence of the files seems to be enough for now. Moved it to the runner though